torntools/changelog.js

```diff
@@ -2,7 +2,7 @@ export default {
 	"v5.7 - xxxxxx, xx. 2021": {
 		Features: ["Add option to export War Report as CSV. - Sashank999"],
 		Fixes: ["Don't show the attack warning on attack logs. - Sashank999", "Fix achievements section not showing. - Sashank999", "Show a warning when you try to train while stacking. - Sashank999",],
-		Changes: ["Only highlight chain timers over a configurable amount. - Sashank999", "Add 'I'm Chaining' button to stop warning for 30 minutes on Attack Page. - Sashank999"],
+		Changes: ["Only highlight chain timers over a configurable amount. - Sashank999", "Add 'I'm Chaining' button to stop warning for 30 minutes on Attack Page. - Sashank999", "Show end time of Wars on faction pages. - Sashank999"],
```

Comment:
Not sure if this would be a feature or a change. I would personally put it as a feature, even though it's a minor one.

Changes are more for changes to already existing TT things.